src/button/ux-button.ts

```diff
@@ -29,12 +29,24 @@ export class UxButton implements Themable {
     if (this.theme) {
       this.styleEngine.applyTheme(this, this.theme);
     }
+
+    if (this.disabled || this.disabled === '') {

```

Comment:
Ahh, yes, that's the correct SO I was looking for. Boolean attributes in HTML are a bit weird about that. I'd def do `setAttibute(`attributeName`, '')`  or all instances of disabled and readonly. I don't think Aurelia is too concerned with XHTML compatibility.